api/nodes.go

```diff
@@ -91,6 +91,23 @@ func (n *Nodes) Stats(nodeID string, q *QueryOptions) (*HostStats, error) {
 	return &resp, nil
 }
 
+func (n *Nodes) GC(nodeID string, q *QueryOptions) error {
+	node, _, err := n.client.Nodes().Info(nodeID, q)
+	if err != nil {
+		return err
+	}
+	if node.HTTPAddr == "" {
+		return fmt.Errorf("http addr of the node %q is running is not advertised", nodeID)
```

Comment:
this message doesn't sound like real english to me? 

`http addr of the node %q is not advertised`  ?